src/assets/i18n/en.json5

```diff
@@ -5390,11 +5390,11 @@
 
   "admin.system-wide-alert.title": "System-wide Alerts",
 
-  "item-access-control-title": "This form allows you to perform changes to the access conditions of the item's metadata or its bitstreams.",
```

Comment:
Please don't remove existing keys from this file.  You've accidentally deleted about 16 different keys from the `en.json5` file.  We cannot merge this PR until they are restored.  You should only add **new keys** and not remove ones that exist (the only exception is if you are replacing a key)